core/dbt/task/runnable.py

```diff
@@ -295,15 +295,15 @@ def _cancel_connections(self, pool):
 
             yellow = dbt.ui.printer.COLOR_FG_YELLOW
             dbt.ui.printer.print_timestamped_line(msg, yellow)
-            raise
 
-        for conn_name in adapter.cancel_open_connections():
-            if self.manifest is not None:
-                node = self.manifest.nodes.get(conn_name)
-                if node is not None and node.is_ephemeral_model:
-                    continue
-            # if we don't have a manifest/don't have a node, print anyway.
-            dbt.ui.printer.print_cancel_line(conn_name)
+        if adapter.cancel_open_connections():
```

Comment:
```suggestion
        else:
```

I think this is ok, no need to do it twice. We should only be calling `cancel_open_connections` in the first place if `is_cancelable` is `True`, and in that case `cancel_open_connections ()` is supposed to return a list.